### PR TITLE
[RA] Add shadow to projectiles.

### PR DESCRIPTION
**Pull request by SoScared**
Adressing #12087 

Played with this the past few months. Simple polish, looks pretty.

As for 25-105mm calibers the shadow provides contrast to the bullet image:
shadowpro

### CODE REVIEW

File: mods/ra/weapons/missiles.yaml
Comment (pchote) [inline mods/ra/weapons/missiles.yaml]: You need to delete the `Shadow: false` 6 lines above this.  This is what causes fort lonestar to fail.

Comment (SoScared) [inline mods/ra/weapons/missiles.yaml]: Done

**Comment by pchote** [comment]
This is going to conflict with the yaml reorganization in #12633, so lets wait until that is merged before fixing this up.

**Comment by ltem** [comment]
For some reason I don't get `APTusk`  weapon (Used by Chrono Tank and Phase Transport) to cast a shadow, at least I don't see it. :/

I'm still looking but I couldn't find an error.

shadow

**Comment by SoScared** [comment]
Perhaps the APTusk missiles has no height to it or the projectile to small to generate a shadow.

**Comment by ltem** [comment]
Yep, that's it. Phase Transport and Chrono Tank have no offset... 
- Link To [Phase Transport](https://github.com/OpenRA/OpenRA/blob/bleed/mods/ra/rules/vehicles.yaml#L717)
- Link To [Chrono Tank](https://github.com/OpenRA/OpenRA/blob/bleed/mods/ra/rules/vehicles.yaml#L652)

**Review by ltem — APPROVED** [review]
Code: No problems found.
Behavior: All projectiles cast shadows
so :+1: 

Little bit out of scope: Phase Transport and Chrono Tank have no offset in the armament, so their attacks won't cast a shadow, should this be fixed in an extra PR?

**Comment by SoScared** [comment]
Huh that's funny. Yeah I'm sure a little height adjustment would look ok.

**Comment by SoScared** [comment]
PR succeeding by #12737 .

**Pull request closed without merging**
